### PR TITLE
Add recommendation text for throttling

### LINKED ISSUE #50: New Connection throttling
Porting https://github.com/w3c/webtransport/issues/230  originally posted to W3C WT @ricea

New connections should be throttled to avoid being a DoS vector. Maybe we will just have some language in the standard to permit and encourage throttling, and leave the details to browsers.

### PR DESCRIPTION
**Pull request by vasilvv**
Closes #50.

### CODE REVIEW

File: draft-ietf-webtrans-http3.md [hunk 1]
```diff
@@ -432,6 +432,10 @@ implement a fairness scheme that ensures that each transport within connection
 gets a reasonable share of controlled resources; this applies both to sending
 data and to opening new streams.
 
+A client may attempt to exhaust resources by opening too many WebTransport
```
Comment (DavidSchinazi) [inline draft-ietf-webtrans-http3.md:435]: Avoid lower-case RFC 2119 keywords:
```suggestion
A client could attempt to exhaust resources by opening too many WebTransport
```

File: draft-ietf-webtrans-http3.md [hunk 2]
```diff
@@ -432,6 +432,10 @@ implement a fairness scheme that ensures that each transport within connection
...
+sessions at once.  In cases when the client is untrusted, the user agent SHOULD
```
Comment (DavidSchinazi) [inline draft-ietf-webtrans-http3.md:436]: I don't understand this, who is the client if there is a user-agent in the picture?

**Review by DavidSchinazi — CHANGES_REQUESTED** [review]

File: draft-ietf-webtrans-http3.md (hunk 2 above)
Comment (vasilvv) [reply draft-ietf-webtrans-http3.md:436]: https://www.ietf.org/archive/id/draft-ietf-webtrans-overview-04.html#name-conventions-and-definitions-2

Comment (DavidSchinazi) [reply draft-ietf-webtrans-http3.md:436]: Ah, thanks.

**Review by DavidSchinazi — APPROVED** [review]

**Review by ekinnear — APPROVED** [review]

**Pull request merged**